examples/getting_started/tf/figs/nvflare_overview.svg

```diff
@@ -0,0 +1 @@
25 3.953125q-2.1875 1.484375 -5.484375 1.484375l-0.484375 0zm32.420013 -0.421875q-4.6875 0 -6.671875 -2.1875q-1.96875 -2.1875 -1.96875 -5.984375l0 -15.671875l-5.8125 0l0 -3.515625l5.8125 0l0 -8.21875l4.265625 0l0 8.21875l9.390625 0l-0.59375 3.515625l-8.796875 0l0 15.40625q0 2.5625 1.171875 3.71875q1.171875 1.140625 3.625 1.140625l4.59375 0l0 3.578125l-5.015625 0zm22.023773 0.421875q-5.59375 0 -8.859375 -3.46875q-3.25 -3.46875 -3.25 -9.75l0 -1.984375q0 -4.109375 1.5 -7.0625q1.5 -2.953125 4.234375 -4.546875q2.75 -1.609375 6.375 -1.609375l0.4375 0q3.5625 0 6.140625 1.5q2.59375 1.484375 4.03125 4.234375q1.453125 2.734375 1.453125 6.421875l0 3.25l-19.953125 0q0.0625 4.75 2.15625 7.09375q2.109375 2.34375 5.734375 2.34375l0.796875 0q2.671875 0 4.34375 -0.953125q1.6875 -0.96875 2.703125 -2.78125l3.25 1.984375q-1.390625 2.453125 -4.03125 3.890625q-2.625 1.4375 -6.265625 1.4375l-0.796875 0zm-7.828125 -16.265625l15.671875 0l0 -0.59375q0 -3.9375 -2.0 -5.953125q-2.0 -2.03125 -5.40625 -2.03125l-0.4375 0q-3.296875 0 -5.46875 2.1875q-2.15625 2.171875 -2.359375 6.390625z" fill-rule="nonzero"/><path fill="#000000" d="m404.49423 586.6236l0 -33.546875l-12.0 0l0 -3.78125l28.421875 0l0 3.78125l-12.0 0l0 33.546875l-4.421875 0zm25.068176 0.640625q-4.375 0 -6.734375 -2.078125q-2.34375 -2.078125 -2.34375 -5.96875q0 -3.203125 2.0 -5.25q2.0 -2.0625 5.890625 -2.59375l10.671875 -1.4375l0 -0.21875q0 -7.296875 -6.5 -7.296875l-0.59375 0q-5.125 0 -6.984375 3.9375l-3.40625 -1.65625q1.21875 -2.765625 3.90625 -4.3125q2.703125 -1.546875 6.484375 -1.546875l0.796875 0q5.171875 0 7.859375 2.71875q2.703125 2.71875 2.703125 7.890625l0 17.171875l-3.890625 0l0 -4.859375q-1.234375 2.5625 -3.734375 4.03125q-2.5 1.46875 -5.65625 1.46875l-0.46875 0zm9.484375 -14.078125l-8.84375 1.28125q-3.09375 0.421875 -4.328125 1.4375q-1.21875 1.015625 -1.21875 3.09375q0 2.296875 1.453125 3.53125q1.46875 1.21875 4.25 1.21875l0.53125 0q3.671875 0 5.90625 -2.34375q2.25 -2.359375 2.25 -6.25l0 -1.96875zm21.972748 14.078125q-8.90625 0 -11.625 -6.234375l3.40625 -1.71875q1.078125 2.25 3.078125 3.3125q2.0 1.0625 5.1875 1.0625l0.59375 0q6.390625 0 6.390625 -4.109375l0 -0.265625q0 -3.5625 -4.203125 -4.09375l-6.078125 -0.859375q-3.578125 -0.484375 -5.53125 -2.296875q-1.9375 -1.8125 -1.9375 -4.90625l0 -0.578125q0 -3.46875 2.765625 -5.59375q2.78125 -2.140625 7.3125 -2.140625l0.6875 0q3.734375 0 6.5 1.515625q2.78125 1.515625 3.84375 4.234375l-3.40625 1.546875q-1.703125 -3.71875 -6.9375 -3.71875l-0.640625 0q-2.765625 0 -4.390625 1.125q-1.625 1.109375 -1.625 2.96875l0 0.328125q0 1.65625 0.984375 2.53125q0.984375 0.875 3.21875 1.203125l6.140625 0.84375q3.515625 0.484375 5.453125 2.46875q1.953125 1.96875 1.953125 5.109375l0 0.578125q0 3.796875 -2.671875 5.75q-2.65625 1.9375 -7.78125 1.9375l-0.6875 0zm17.597076 -0.640625l0 -38.921875l4.265625 0l0 24.046875l13.4375 -12.265625l5.609375 0l-11.359375 10.28125l12.203125 16.859375l-5.171875 0l-10.1875 -14.234375l-4.53125 4.109375l0 10.125l-4.265625 0z" fill-rule="nonzero"/></g></svg>
```

Comment:
Maybe we should move these figures to a central place, e.g. in docs and refer to here rather than duplicating them. @chesterxgchen

Reply:
sure, moving to `doc/resources` would work